src/eslint/types.d.ts

```diff
@@ -21,3 +21,9 @@ declare module "@next/eslint-plugin-next" {
     };
     export const rules: Record<string, Rule.RuleModule>;
 }
+
+declare module "eslint-config-expo/flat" {
+    import type { Linter, Rule } from "eslint";
+
+    export default expoConfig = [] as Linter.Config[];
+}
```

Comment:
I don't believe this is needed. You can drop this bit and also remove the `/// <reference types="./types.d.ts" />` on top of the `expo.js` file.